lib/fhir_client/sections/crud.rb

```diff
@@ -117,8 +117,8 @@ def destroy(klass, id = nil, options = {})
       # Create a new resource with a server assigned id. Return the newly created
       # resource with the id the server assigned.
       #
-      def create(resource, format = @default_format)
-        base_create(resource, nil, format)
+      def create(resource, options = {}, format = @default_format)
```

Comment:
small remark: if there are users of this gem, passing the format, this will be a breaking change (good to add it to the changelog)

Reply:
Good point. We can probably solve that by swapping the order of the parameters, by putting `options` last.

Reply:
Or maybe it isn't that simple.